src/commands/flinkComputePools.ts

```diff
@@ -60,6 +65,36 @@ export async function selectPoolForArtifactsViewCommand(item?: CCloudFlinkComput
   commands.executeCommand("confluent-flink-artifacts.focus");
 }
 
+/**
+ * Show a quickpick to select a default {@link FlinkComputePool} and database
+ * for Flink SQL operations.
+ */
+export async function configureFlinkDefaults() {
+  const config: Record<string, any> = await workspace.getConfiguration("confluent");
+  const flinkConfig = config["flink"] || {};
+
+  const computePool = await flinkComputePoolQuickPick();
```

Comment:
We kinda want to be able to have the quickpick be able to default to the current default set one, otherwise the currently viewed one. That logic probably needs to be done outside of the call to `flinkComputePoolQuickPick()`, 'cause other times we may well want the _opposite_ defaulting hierarchy I think.

Can tackle in subsequent ticket, but noting here right now.